<launch>
    <arg name="dataset"     default="dataset_sim" />
    <arg name="bag"         default="/home/junlin/$(arg dataset).bag" />

    <node name="projection_test" pkg="ov_msckf" type="projection_test" output="screen" clear_params="true" required="true">
        <param name="path_bag"        type="str" value="$(arg bag)" />
    </node>
    <!-- <node type="rviz" name="rviz" pkg="rviz" args="-d $(find ov_msckf)/launch/display_cse.rviz" /> -->
    
</launch>
